spec/features/create_todo_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe "Todo items" do
+   before do
+      @user = create(:user)
+      visit(user_path(@user))
+    end
+  
+  describe "Creating todo items" do
+   
+    it "allows user to create todo item" do
+      expect{ @user.items.create(title: 'new task') }.to change{ @user.items.count }.by(1) 
+    end
+
```

Comment:
Here I would like to test using filling in form field then pressing enter to create todo_item